cts/cts-cli.in

```diff
@@ -1276,7 +1276,7 @@ class CibadminRegressionTest(RegressionTest):
 
         # Add some stuff to the empty CIB so we know that erasing it did something.
         basic_tests_setup = [
-            """cibadmin -C -o nodes --xml-text '<node id="node1" uname="node1">'""",
+            """cibadmin -C -o nodes --xml-text '<node id="node1" uname="node1"/>'""",
```

Comment:
I'm surprised the outputs don't change... I would've guessed an error on first parse and then success on second parse. Oh well.

Reply:
There are two aspects to the strict interpretation: accepting partial XML and accepting recoverably invalid XML. The retry was for the second category, but this one fell into the first. If we stopped accepting partial XML (only), it *would* be detected and recovered as invalid XML, but accepting the partial XML meant the recovery never kicked in.